### PR TITLE
Pages

### PR DESCRIPTION
**Pull request by mansipadgelwar**
Add pages for Ecomm App
Pages added:
1. Home Page
2. Product Listing Page
3. My Wishlist
4. My Cart

### CODE REVIEW

File: src/pages/Authentication/Login/Login.jsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+import { Link } from "react-router-dom";
+
+const Login = () => {
+  return (
+    <div className="authentication-page">
+      <Header />
+      <article className="form-container login-form">
+        <div className="authentication-form-container">
+          <form className="form" action="">
+            <h3 className="h3">LOGIN</h3>
```
Comment (manish591) [inline src/pages/Authentication/Login/Login.jsx:9]: Don't give class names the same as elements.

File: src/pages/Authentication/Login/Login.jsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+              <input
+                className="input"
+                type="email"
+                id="email"
+                name="email"
+                placeholder="Enter Email ID"
+              />
+            </div>
+
+            <label for="password" className="input-label">
```
Comment (manish591) [inline src/pages/Authentication/Login/Login.jsx]: Change `for` to `htmlFor`.

**Review by manish591 — CHANGES_REQUESTED** [review]
Your code is clean and readable also, 
I saw you are importing Header on every page instead you can import it inside app.js and put it above all the routers so that it will be visible for all the pages.

**Review by manish591 — APPROVED** [review]

File: src/components/VerticalCard/VerticalCard.js [hunk 3]
```diff
@@ -3,7 +3,7 @@ const VerticalCard = () => {
     <article className="vertical-card">
       <span className="material-icons-outlined icon">favorite</span>
       <div className="img-container">
-        <img src="/images/Plant1.jpg" alt="plant" className="img-responsive" />
```
Comment (shilpe26) [inline src/components/VerticalCard/VerticalCard.js:6]: Good you have applied alt attribute to the images

File: src/pages/Authentication/Signup/Signup.jsx [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+              class="input"
+              type="email"
+              id="email"
+              name="email"
+              placeholder="Enter your email ID"
+              required
+            />
+
+            <label htmlFor="fname" class="input-label">
+              First Name:{" "}
```
Comment (shilpe26) [inline src/pages/Authentication/Signup/Signup.jsx]: remove these unwanted braces

**Review by shilpe26 — APPROVED** [review]
The file is easy to read as you applied prettier 👍
The file structuring is good
good to merge after seeing the suggestion given

**Pull request merged**